.github/workflows/build.yml

```
@@ -70,9 +74,11 @@ jobs:
           echo "Releasename: ${releasename}"
           echo "GH_DEPLOY_ENVIRONMENT=pr-${branchname}" >> $GITHUB_ENV
           echo "RELEASENAME=${releasename}" >> $GITHUB_ENV
-      - name: Get release name
+      - name: Get deployment name for master branch
```

```suggestion
      - name: Set deployment name for master branch
```
Very very small thing I know :)

---

Will leave it as is, as all the other steps are named accordingly